src/libs/Notification/PushNotification/NotificationType.ts

```diff
@@ -0,0 +1,28 @@
+import {OnyxServerUpdate} from '@src/types/onyx/OnyxUpdatesFromServer';
+
+const NotificationType = {
+    REPORT_COMMENT: 'reportComment',
+} as const;
+
+type NotificationDataMap = {
+    [NotificationType.REPORT_COMMENT]: ReportCommentNotificationData;
+};
+
+type NotificationData = ReportCommentNotificationData;
```

Comment:
Is that necessary?

Reply:
I created this `NotificationData` thinking on the future, at the moment we have only one notification type, but if we have more it would be more easier to add to the code and this type 🙂